notebooks/mos_spectroscopy_advanced/MOSspec_advanced.ipynb

```diff
@@ -0,0 +1,1060 @@
+{
```

Comment:
<p>This might have already been addressed, as you mentioned. I have yet to encounter one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/117/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I do not see this anymore. Must have been fixed!</p>